apps/swirl-docs/src/lib/docs/src/oasBuilder.ts

```diff
@@ -332,45 +336,25 @@ export default class OASBuilder implements IOASBuilder {
   } {
     const har = OASToHar(this.oas, operation);
     const harRequest = har.log.entries[0].request;
+    const body = operation.getRequestBodyExamples()[0]?.examples[0]?.value;
 
     return {
-      snippets: {
-        ocaml: String(oasToSnippet(this.oas, operation, {}, {}, "ocaml").code),
-        c: String(oasToSnippet(this.oas, operation, {}, {}, "c").code),
-        csharp: String(
-          oasToSnippet(this.oas, operation, {}, {}, "csharp").code
-        ),
-        go: String(oasToSnippet(this.oas, operation, {}, {}, "go").code),
-        java: String(oasToSnippet(this.oas, operation, {}, {}, "java").code),
-        javascript: String(
-          oasToSnippet(this.oas, operation, {}, {}, "javascript").code
-        ),
-        kotlin: String(
-          oasToSnippet(this.oas, operation, {}, {}, "kotlin").code
-        ),
-        node: String(oasToSnippet(this.oas, operation, {}, {}, "node").code),
-        php: String(oasToSnippet(this.oas, operation, {}, {}, "php").code),
-        python: String(
-          oasToSnippet(this.oas, operation, {}, {}, "python").code
-        ),
-        ruby: String(oasToSnippet(this.oas, operation, {}, {}, "ruby").code),
-        shell: String(oasToSnippet(this.oas, operation, {}, {}, "shell").code),
-        swift: String(oasToSnippet(this.oas, operation, {}, {}, "swift").code),
-        http: String(oasToSnippet(this.oas, operation, {}, {}, "http").code),
-        clojure: String(
-          oasToSnippet(this.oas, operation, {}, {}, "clojure").code
-        ),
-        cplusplus: String(
-          oasToSnippet(this.oas, operation, {}, {}, "cplusplus").code
-        ),
-        objectivec: String(
-          oasToSnippet(this.oas, operation, {}, {}, "objectivec").code
-        ),
-        powershell: String(
-          oasToSnippet(this.oas, operation, {}, {}, "powershell").code
-        ),
-        r: String(oasToSnippet(this.oas, operation, {}, {}, "r").code),
-      },
+      snippets: Object.fromEntries(
+        (Object.keys(supportedLanguages) as SupportedTargets[]).map((l) => [
```

Comment:
**nitpick (blocking):** we currently don't have documented coding styles like this, but i prefer using readable names for variables. so l should be language here :)